plugins/services/src/js/constants/OperatorTypes.js

```diff
@@ -2,37 +2,51 @@ const OperatorTypes = {
   UNIQUE: {
     requiresValue: false,
     requiresEmptyValue: true,
-    stringNumberValue: false
+    stringNumberValue: false,
+    name: "Unique",
+    description: "Run each app task on a unique attribue id"
   },
   CLUSTER: {
     requiresValue: true,
     requiresEmptyValue: false,
-    stringNumberValue: false
+    stringNumberValue: false,
+    name: "Cluster",
+    description: "Run app tasks on nodes that share a certain attribute id"
   },
   GROUP_BY: {
     requiresValue: false,
     requiresEmptyValue: false,
-    stringNumberValue: true
+    stringNumberValue: true,
+    name: "Group By",
+    description: "Run app tasks evenly distributed across a certain attribute"
   },
   IS: {
     requiresValue: true,
     requiresEmptyValue: false,
-    stringNumberValue: false
+    stringNumberValue: false,
+    name: "Is",
```

Comment:
🤔  Is this suppose to be the same description as "cluster"?

Reply:
good catch! probably not, but thats all we have (see screenshot by dima above), can you maybe check back with lily for a new text?

Reply:
We will have to follow-up with someone from docs on this. The other descriptions came from here: http://mesosphere.github.io/marathon/docs/constraints.html, but IS is not on that page so we don't have a good idea what the description is.

Reply:
I suggest not blocking this PR, and make a small follow-up ticket to improve this description.

Reply:
@juliangieseke @ahoskins did one of you file an issue?